l10n_de_website_sale_accept_terms/i18n/de.po

```diff
@@ -0,0 +1,179 @@
+# Translation of Odoo Server.
```

Comment:
not a empty file - add a empty translation file like:
https://github.com/OCA/l10n-germany/blob/7.0/l10n_de_country_states/i18n/l10n_de_country_states.po